swarm/storage/feed/handler.go

```diff
@@ -94,8 +93,8 @@ func (h *Handler) Validate(chunkAddr storage.Address, data []byte) bool {
 
 	// First, deserialize the chunk
 	var r Request
-	if err := r.fromChunk(chunkAddr, data); err != nil {
-		log.Debug("Invalid feed update chunk", "addr", chunkAddr.Hex(), "err", err.Error())
+	if err := r.fromChunk(chunk); err != nil {
+		log.Debug("Invalid feed update chunk", "addr", chunk.Address().Hex(), "err", err.Error())
```

Comment:
`chunk.Address()` and `err` are enough for the logger

Reply:
Corrected, thanks.